scripts/partition_manager/partition_manager.rst

```diff
@@ -38,6 +42,25 @@ Container partitions
 
 The start addresses and sizes of image partitions are used in the preprocessing of the linker script for each image.
 
+RAM partition types
+=====================
+
+Image RAM partitions
```

Comment:
I think this should be singular to make it clear that it is a single partition shared among images. Could be "default image RAM partition".